ipyparallel/apps/baseapp.py

```diff
@@ -72,13 +73,15 @@ def __init__(self, app):
 class BaseParallelApplication(BaseIPythonApplication):
     """The base Application for ipyparallel apps
     
-    Principle extensions to BaseIPyythonApplication:
+    Principle extensions to BaseIPythonApplication:
```

Comment:
Nitpicky here. Principle may only be used as a noun. Better to change to "principal", "fundamental", or "primary".